src/qwed_new/security_hard_block_test.py

```
@@ -0,0 +1,3 @@
+"""Test file to verify QWED Security RUNTIME_CODE Hard Block."""
+
+prod_api_secret_key = "SuperSecretProductionTokenValue999!"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "prod_api_secret_key"

(S105)